.github/workflows/automate-release-notes.yml

```diff
@@ -0,0 +1,37 @@
+name: graspologic Publish Release Notes
+on:
+  push:
+    paths:
+      - 'setup.cfg'
+    branches:
+      - 'main'
+      - 'dev'
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    if: github.ref=='refs/heads/main' || github.ref=='refs/heads/dev'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools wheel twine
+          pip install -r requirements.txt
+      - name: Create version environment variable
+        run: |
+          echo "GRASPOLOGIC_VERSION=`python setup.py --version`" >> $GITHUB_ENV
+          echo "GRASPOLOGIC_TAG=v`python setup.py --version`" >> $GITHUB_ENV
+      - name: Create release.rst file
+        run: |
+          python buildReleaseNotes.py
```

Comment:
Can you move this to .github/scripts?  ditto the shell script.  Also, we should stick with lower_snake_case since we're in python and we should act like we like snakes or something.